react/Modal/index.jsx

```diff
@@ -158,7 +161,12 @@ const ModalHeader = ({
 }) => {
   const isTitle = typeof children === 'string'
   return (
-    <div className={cx(styles['c-modal-header'], className)} style={style}>
+    <div
+      className={cx(styles['c-modal-header'], className)}
+      style={style}
+      id={titleId}
+      tabIndex="-1"
```

Comment:
`tabIndex="-1"` is to make sure that when the modal takes the focus, reading began at the beginning of the modal and not the next focusable element (inside the modal) which is the behaviour by default.

Reply:
Why do we have to put an id ?

Reply:
In order to assign a title to the modal, having a `<hn>` isn't always enough apparently.

Reply:
Assigning a title, what for ?

Reply:
It's a label, to be read, so the reader knows what the content of the modal is all about.

Reply:
There's `aria-labelledBy={titleId}` on the Modal, L320
I didn't add `aria-describedby` as it is optional, contextual and in many case, not relevant.
As for the unique ID being generated outside the modal, if I generate it inside the Modal, how can `ModalHeader` can use it too then?

Reply:
I did not understand that aria-labelledBy was a reference and not the text to be read.

`titleId` should be generated in the Modal render and passed to ModalHeader though.